{
  "seleniumLog": [],
  "checkPoints": [
    {
      "message": "Exception authenticating MongoCredential{mechanism=SCRAM-SHA-1, userName='support', source='RDM', password=<hidden>, mechanismProperties=<hidden>}",
      "type": "Fail",
      "screenshot": "",
      "duration": 0,
      "threshold": 0,
      "subCheckPoints": []
    }
  ],
  "errorTrace": "com.mongodb.MongoSecurityException: Exception authenticating MongoCredential{mechanism=SCRAM-SHA-1, userName='support', source='RDM', password=<hidden>, mechanismProperties=<hidden>}\r\n\tat com.mongodb.internal.connection.SaslAuthenticator.wrapException(SaslAuthenticator.java:201)\r\n\tat com.mongodb.internal.connection.SaslAuthenticator.access$300(SaslAuthenticator.java:40)\r\n\tat com.mongodb.internal.connection.SaslAuthenticator$1.run(SaslAuthenticator.java:78)\r\n\tat com.mongodb.internal.connection.SaslAuthenticator$1.run(SaslAuthenticator.java:47)\r\n\tat com.mongodb.internal.connection.SaslAuthenticator.doAsSubject(SaslAuthenticator.java:207)\r\n\tat com.mongodb.internal.connection.SaslAuthenticator.authenticate(SaslAuthenticator.java:47)\r\n\tat com.mongodb.internal.connection.InternalStreamConnectionInitializer.authenticateAll(InternalStreamConnectionInitializer.java:156)\r\n\tat com.mongodb.internal.connection.InternalStreamConnectionInitializer.initialize(InternalStreamConnectionInitializer.java:63)\r\n\tat com.mongodb.internal.connection.InternalStreamConnection.open(InternalStreamConnection.java:129)\r\n\tat com.mongodb.internal.connection.UsageTrackingInternalConnection.open(UsageTrackingInternalConnection.java:50)\r\n\tat com.mongodb.internal.connection.DefaultConnectionPool$PooledConnection.open(DefaultConnectionPool.java:398)\r\n\tat com.mongodb.internal.connection.DefaultConnectionPool.get(DefaultConnectionPool.java:115)\r\n\tat com.mongodb.internal.connection.DefaultConnectionPool.get(DefaultConnectionPool.java:101)\r\n\tat com.mongodb.internal.connection.DefaultServer.getConnection(DefaultServer.java:92)\r\n\tat com.mongodb.binding.ClusterBinding$ClusterBindingConnectionSource.getConnection(ClusterBinding.java:126)\r\n\tat com.mongodb.operation.ListCollectionsOperation$1.call(ListCollectionsOperation.java:234)\r\n\tat com.mongodb.operation.ListCollectionsOperation$1.call(ListCollectionsOperation.java:231)\r\n\tat com.mongodb.operation.OperationHelper.withReadConnectionSource(OperationHelper.java:463)\r\n\tat com.mongodb.operation.ListCollectionsOperation.execute(ListCollectionsOperation.java:231)\r\n\tat com.mongodb.operation.ListCollectionsOperation.execute(ListCollectionsOperation.java:86)\r\n\tat com.mongodb.client.internal.MongoClientDelegate$DelegateOperationExecutor.execute(MongoClientDelegate.java:196)\r\n\tat com.mongodb.client.internal.MongoIterableImpl.execute(MongoIterableImpl.java:143)\r\n\tat com.mongodb.client.internal.MongoIterableImpl.iterator(MongoIterableImpl.java:92)\r\n\tat com.mongodb.client.internal.MappingIterable.iterator(MappingIterable.java:39)\r\n\tat com.mongodb.client.internal.MappingIterable.iterator(MappingIterable.java:27)\r\n\tat com.testscripts.Tests.f(Tests.java:49)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.lang.reflect.Method.invoke(Unknown Source)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:645)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:778)\r\n\tat org.testng.TestRunner.run(TestRunner.java:632)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:387)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:289)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1218)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1133)\r\n\tat org.testng.TestNG.run(TestNG.java:1104)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\nCaused by: com.mongodb.MongoCommandException: Command failed with error 18 (AuthenticationFailed): 'Authentication failed.' on server hfsuatvumgdb01:27019. The full response is {\"ok\": 0.0, \"errmsg\": \"Authentication failed.\", \"code\": 18, \"codeName\": \"AuthenticationFailed\"}\r\n\tat com.mongodb.internal.connection.ProtocolHelper.getCommandFailureException(ProtocolHelper.java:175)\r\n\tat com.mongodb.internal.connection.InternalStreamConnection.receiveCommandMessageResponse(InternalStreamConnection.java:303)\r\n\tat com.mongodb.internal.connection.InternalStreamConnection.sendAndReceive(InternalStreamConnection.java:259)\r\n\tat com.mongodb.internal.connection.CommandHelper.sendAndReceive(CommandHelper.java:83)\r\n\tat com.mongodb.internal.connection.CommandHelper.executeCommand(CommandHelper.java:33)\r\n\tat com.mongodb.internal.connection.SaslAuthenticator.sendSaslStart(SaslAuthenticator.java:158)\r\n\tat com.mongodb.internal.connection.SaslAuthenticator.access$100(SaslAuthenticator.java:40)\r\n\tat com.mongodb.internal.connection.SaslAuthenticator$1.run(SaslAuthenticator.java:54)\r\n\t... 48 more\r\n"
}